ibm/mas_devops/roles/kubeturbo/README.md

```diff
@@ -0,0 +1,81 @@
+Kubeturbo
+============
+Installs a kubeturbo instance and sends data to a defined Turbonomic instance under the cluster name.
+
+Role Variables
+--------------
+
+### cluster_name
+Required. The cluster name is required to install Kubeturbo.
```

Comment:
This needs more description. From the previous review the cluster_name was the OCP cluster we are monitoring that kubeturbo is deployed on I believe. The docs need to say that.